src/components/Landing.js

```diff
@@ -4,15 +4,18 @@ import '../assets/buttons.css';
 import './Landing.css';
 
 class Landing extends React.Component {
-  constructor () {
+  constructor ({ match }) {
     super();
+    let _hideArtwork = match.params.hideArtwork === "true";
```

Comment:
Why not just calling the variable `hideArtwork`?

Reply:
- Use single quotations marks (`"true"`)
- Use `const` instead of `let`

Reply:
- `hideArtwork` is a string, triggered when the word `'hard'` is present in the route, mainly for easier user navigation (the user could remove `/hard` from the url and find themselves in the regular mode), I find it useful to give meaningful-human-readable routes.

Reply:
Good thing you spotted it, it's an old code, back when I used routing params for the `Landing` comonent